### PR TITLE
New entries

### PR DESCRIPTION
**Pull request by msalerno2**

### CODE REVIEW

File: literatur.bib [hunk 1]
```diff
@@ -2675,6 +2675,13 @@ @Proceedings{celorrio-et-al-2011
   year =         "2011"
 }
 
+@InProceedings{celorrio-et-al-icaps2013wspal,
+  author =       "Sergio J. Celorrio and Patrick Haslum and Sylvie Thiebaux",
```
Comment (jendrikseipp) [inline literatur.bib]: Add accent to Sylvie's last name (see other entries).

File: literatur.bib [hunk 2]
```diff
@@ -2675,6 +2675,13 @@ @Proceedings{celorrio-et-al-2011
...
+  title =        "Pruning bad quality causal links in sequential satisfying planning",
```
Comment (jendrikseipp) [inline literatur.bib:2681]: Use title case (except if the title in the PDF really only uses lower case). Here and for other entries.

File: literatur.bib [hunk 3]
```diff
@@ -5856,6 +5877,14 @@ @TechReport{fink-veloso-tr1994
   number =       "CMU-CS-94-123"
 }
 
+@InProceedings{fink-yang-cscsi1992,
+  author =       "Eugene Fink and Qiang Yang",
+  title =        "Formalizing plan justifications",
+  booktitle =    "ProceedingsProceeding of the Ninth Conference of the Society for Computational Studies of Intelligence",
```
Comment (jendrikseipp) [inline literatur.bib]:
```suggestion
  booktitle =    "Proceedings of the Ninth Conference of the Society for Computational Studies of Intelligence",
```

File: literatur.bib [hunk 4]
```diff
@@ -10933,6 +10962,17 @@ @Book{lawler-1976
   year =         "1976"
 }
 
+@Article{leberre-parrain-jsbmc2010,
+  title =        "The Sat4j library, release 2.2",
```
Comment (jendrikseipp) [inline literatur.bib]:
```suggestion
  title =        "The {Sat4j} library, release 2.2",
```

File: literatur.bib [hunk 5]
```diff
@@ -10933,6 +10962,17 @@ @Book{lawler-1976
...
+  author =       "Daniel Le Berre and Anne Parrain",
```
Comment (jendrikseipp) [inline literatur.bib]:
```suggestion
  author =       "Daniel {Le Berre} and Anne Parrain",
```

File: literatur.bib [hunk 6]
```diff
@@ -12445,6 +12517,13 @@ @Misc{olmo-et-al-arxiv2021
   howpublished = "arXiv:2106.07131 [cs.CL]"
 }
 
+@InProceedings{olz-bercher-icaps2019,
+  author =       "Conny Olz and Pascal Bercher",
+  title =        "Eliminating Redundant Actions in Partially Ordered Plans -- A Complexity Analysis",
```
Comment (jendrikseipp) [inline literatur.bib:12522]:
```suggestion
  title =        "Eliminating Redundant Actions in Partially Ordered Plans --- A Complexity Analysis",
```

File: literatur.bib [hunk 7]
```diff
@@ -15007,6 +15093,15 @@ @InProceedings{shvo-et-al-cai2018
   pages =        "253--258"
 }
 
+@Article{siddiqui-haslum-jair2015,
+  author =       "Fazlul Hasan Siddiqui and Patrick Haslum",
```
Comment (jendrikseipp) [inline literatur.bib]:
```suggestion
  author =       "Fazlul Hasan Siddiqui and Patrik Haslum",
```

**Review by jendrikseipp — CHANGES_REQUESTED** [review]

File: literatur.bib (hunk 1 above)
Comment (maltehelmert) [reply literatur.bib]: There is a also a typo in Patrik Haslum's name.

Comment (maltehelmert) [reply literatur.bib]: (Also for at least one more entry -- I didn't look at the whole diff, only at the parts Jendrik answered to.)

File: literatur.bib [hunk 8]
```diff
@@ -10933,6 +10962,17 @@ @Book{lawler-1976
   year =         "1976"
 }
 
+@Article{leberre-parrain-jsbmc2010,
+  title =        "The Sat4j library, release 2.2",
+  author =       "Daniel Le Berre and Anne Parrain",
+  journal =      "Journal on Satisfiability, Boolean Modeling and Computation",
+  volume =       "7",
+  number =       "2-3",
```
Comment (maltehelmert) [inline literatur.bib]: Should be "--" instead of "-".

File: literatur.bib [hunk 9]
```diff
@@ -11734,6 +11774,15 @@ @InProceedings{mausam-weld-aaai2004
...
 
+@InProceedings{mcallester-rosenblitt-tr1991,
+  author =       "David A. McAllester and David Rosenblitt",
+  title =        "Systematic Nonlinear Planning",
+  booktitle =    "Proceedings of the 9th National Conference on Artificial Intelligence, Volume 2",
+  publisher =    "{AAAI} Press",
+  year =         "1991",
+  pages =        "634--639",
+}
+
```
Comment (maltehelmert) [inline literatur.bib:11785]: I think this entry needs to be removed or redone. If a volume is given, it should be in the volume field. The key says it's a TR, but the description says it's a AAAI paper (but it doesn't follow our conventions for AAAI papers). And there shouldn't be a comma at the end.

File: literatur.bib [hunk 10]
```diff
@@ -12144,6 +12200,15 @@ @InProceedings{muise-et-al-icaps2022systemdemos
   crossref =     "icaps2022systemdemos"
 }
 
+@Article{muise-et-al-jair2016,
+  author =       "Christian Muise and J. Christopher Beck and Sheila A. McIlraith",
+  title =        "Optimal partial-order plan relaxation via {M}ax{SAT}",
```
Comment (maltehelmert) [inline literatur.bib]: Using {...} like this is not strictly speaking wrong, but I would write it as {MaxSAT} instead. Same result, less syntax and works much more nicely with spellcheckers.

File: literatur.bib [hunk 11]
```diff
@@ -11734,6 +11774,15 @@ @InProceedings{mausam-weld-aaai2004
...
+  author =       "David A. McAllester and David Rosenblitt",
+  title =        "Systematic Nonlinear Planning",
+  booktitle =    "Proceedings of the 9th National Conference on Artificial Intelligence, Volume 2",
+  publisher =    "{AAAI} Press",
+  year =         "1991",
+  pages =        "634--639",
+}
+
 @Misc{mccarthy-1963,
   author =       "John Mc{C}arthy",
```
Comment (maltehelmert) [inline literatur.bib:11787]: Shouldn't have "{...}". bibtex doesn't mess with last names.

Comment (maltehelmert) [reply literatur.bib:11787]: ...but that wasn't your change, oops! :-)

File: literatur.bib (hunk 6 above)
Comment (maltehelmert) [reply literatur.bib:12522]: That's debatable. I think many modern styles prefer "--" over "---" everywhere.

Comment (jendrikseipp) [reply literatur.bib:12522]: Agreed. But the paper uses three dashes :)

File: literatur.bib (hunk 9 above)
Comment (msalerno2) [reply literatur.bib:11785]: The [dblp](https://dblp.org/rec/conf/aaai/McAllesterR91.html?view=bibtex) entry says that, but I've seen some papers cite it as a technical report, while others cite it as a AAAI paper. So I'm not completely sure how to create this entry. I'll remove it for now.

Comment (abcorrea) [reply literatur.bib:11785]: It should be: 

```bibtex
@InProceedings{mcallester-rosenblitt-aaai1991,
  author =       "David A. McAllester and David Rosenblitt",
  title =        "Systematic Nonlinear Planning",
  crossref =     aaai1991,
  volume =       "2",
  pages =        "634--639",
}
```

and the cross-references for `aaai1991` should be added accordingly.

(I am not sure if the `A.` in McAllester's name should be there, as it is not in the paper. However, Russell and Norvig do have the middle initial `A.`)

File: literatur.bib (hunk 6 above)
Comment (maltehelmert) [reply literatur.bib:12522]: That doesn't really matter, this is the kind of typesetting convention that isn't supposed to be "exported" from the original title to the citing paper, but the conventions of the citing paper apply. (In other words, the goal should be consistency on this within the bib file.) Not that I think we're currently 100% consistent.

Comment (jendrikseipp) [reply literatur.bib:12522]: Good to know, I didn't have that on my radar.

File: crossref.bib [hunk 12]
```diff
@@ -7,6 +7,15 @@
 %
 % Do not cite directly. Instead, add a Book entry in literatur.bib.
 %
+@Proceedings{aaai1991,
+  title =        "Proceedings of the Ninth National Conference on
+                  Artificial Intelligence ({AAAI} 1991)",
+  booktitle =    "Proceedings of the Eleventh National Conference on
```
Comment (jendrikseipp) [inline crossref.bib]:
```suggestion
  booktitle =    "Proceedings of the Ninth National Conference on
```

File: crossref.bib [hunk 13]
```diff
@@ -7,6 +7,15 @@
...
+                  Artificial Intelligence ({AAAI} 1991)",
+  publisher =    "{AAAI} Press/{MIT} Press",
```
Comment (jendrikseipp) [inline crossref.bib]:
```suggestion
  publisher =    "{AAAI} Press",
```
see https://dl.acm.org/doi/proceedings/10.5555/1865675

File: literatur.bib [hunk 14]
```diff
@@ -2675,6 +2675,13 @@ @Proceedings{celorrio-et-al-2011
   year =         "2011"
 }
 
+@InProceedings{celorrio-et-al-icaps2013wspal,
+  author =       "Sergio Jim{\'{e}}nez Celorrio and Patrik Haslum and Sylvie Thi{\'e}baux",
```
Comment (jendrikseipp) [inline literatur.bib]:
```suggestion
  author =       "Sergio {Jim{\'e}nez Celorrio} and Patrik Haslum and Sylvie Thi{\'e}baux",
```

File: literatur.bib (hunk 2 above)
Comment (jendrikseipp) [reply literatur.bib:2681]: Please add `% Title uses lower case.` above the entry.

File: literatur.bib [hunk 15]
```diff
@@ -12445,6 +12516,13 @@ @Misc{olmo-et-al-arxiv2021
   howpublished = "arXiv:2106.07131 [cs.CL]"
 }
 
+@InProceedings{olz-bercher-icaps2019,
+  author =       "Conny Olz and Pascal Bercher",
+  title =        "Eliminating Redundant Actions in Partially Ordered Plans --- A Complexity Analysis",
```
Comment (jendrikseipp) [inline literatur.bib]:
```suggestion
  title =        "Eliminating Redundant Actions in Partially Ordered Plans -- A Complexity Analysis",
```

**Review by jendrikseipp — CHANGES_REQUESTED** [review]

**Pull request merged**